packages/engineer/src/feature/gui.dev-server.env.ts

```diff
@@ -87,6 +90,7 @@ function createDashboardConfig({
                 chunks: ['index'],
                 title,
                 favicon,
+                meta: htmlMeta ?? {},
```

Comment:
Why the fallback object?

Reply:
If it's empty, the default value is `{}`
but if htmlMeta is `undefined`, webpack doesn't know how to handle it
`'Html Webpack Plugin:\n  TypeError: Cannot convert undefined or null to object\n`
@AviVahl